app/templates/main.html

```diff
@@ -57,18 +56,6 @@ <h1>Able </h1>
       <span id="place-name"  class="title"></span><br>
       <span id="place-address"></span>
     </div>
-    <div id="sidenav">
-
-        <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
-        <a href="/">Home</a>
-        <a href="#">About</a>
-        <a href="#">Profile</a>
-        <a href="navigation">Navigation</a>
-        <a href="#">Contact Us</a>
-        <a href="#">Business Profile</a>
-        <a href="#">Change Language</a>
-
-    </div>
     <script>
```

Comment:
This whole script block needs to be taken out.

Reply:
Move this code to a new JS file or remove. I think this is Isha's code she sent you, right?